samples/features/sql-big-data-cluster/deployment/private-aks/scripts/deploy-private-aks-udr.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
+
+#Get Subscription ID and service principles as input. It is used as default for controller, SQL Server Master instance (sa account) and Knox.
+
+while true; do
+    read -s -p "Your Azure Subscription: " subscription
+    echo
+    read -s -p "Your service principles ID: " sp_id
+    echo
+    read -s -p "Your service principles Password: " sp_pwd
+
+done
+
+#Define a set of environment variables to be used in resource creations.
+
+export SUBID=$subscription
+
+export REGION_NAME=northeurope
```

Comment:
Region and resource group name should not have defaults. A customer from US running this script would not want to use northeurope region.